rootfs/Dockerfile

```
@@ -15,6 +15,7 @@ RUN mkdir -p /mnt/sys-root; \
     gzip \
     hwdata \
     iptables-libs \
+    iproute \
```

```suggestion
    iproute \
    iptables-libs \
```
To respect alphabetical order